### ISSUE TITLE
Guard against redefining input

### ISSUE THREAD

**Issue by taylorzr** [taylorzr/verbalize#23]
Now that we have inputs introspection, we might want some guard against accidentally calling input a second time. E.g.
```ruby
class Add
  input :a, optional: :b
end

Add.inputs # => [:a, :b]
Add.input # Oops, I've just redefined the action's inputs and internal metaprogrammed methods
Add.inputs # => []
```

Thoughts @tmertens @Kuraiou ?

**Comment by taylorzr** [comment]
Or maybe this input/inputs is confusing and we should move towards something like input= vs inputs.

**Comment by Kuraiou** [comment]
I think it's okay to leave it: you may want to do some magic with
dynamically assigning inputs which is now actually possible, but raising
errors would block that behavior.

**Comment by taylorzr** [comment]
https://github.com/taylorzr/verbalize/pull/27

**Comment by taylorzr** [comment]
Fixed by #27
